GVFS/GVFS.Common/PlaceholderListDatabase.cs

```diff
@@ -371,6 +377,19 @@ public bool IsExpandedFolder
                     return this.Sha == ExpandedFolderValue;
                 }
             }
+
+            public bool IsPossibleTombstoneFolder
+            {
+                get
+                {
+                    return this.Sha == PossibleTombstoneFolderValue;

```

Comment:
While not something this project seems to always follow, I like when we explicitly define which string comparison strategy to use instead of the `==` method.

What string comparison strategy is most appropriate here? (Ordinal Case Sensitive)? Same for `IsShaAFolder`

Reply:
Yeah I was following what the code in this file was already doing so I will leave this for another PR.  Do you know what the performance difference in using `==` vs using a comparison strategy would be?

Reply:
> Do you know what the performance difference in using == vs using a comparison strategy would be?

I think the `==` operator implicity [calls the static Equals(String, String) method](https://docs.microsoft.com/en-us/dotnet/api/system.string.equals?view=netframework-4.7.2#System_String_Equals_System_String_System_String_),  which [performs an ordinal (case-sensitive and culture-insensitive) comparison.](https://docs.microsoft.com/en-us/dotnet/api/system.string.equals?view=netframework-4.7.2#System_String_Equals_System_String_System_String_)

TLDR: I think the equality operator is using the string comparison method we want, so there shouldn't be any perf difference. The only difference would be that it would be explicitly obvious in code and force developers to make an explicit choice when writing code.